internal/webui/public/index.html

```diff
@@ -105,21 +105,21 @@ <h1>Welcome to the Installer!</h1>
 
               codeMirrorEditor.on('change', () => content = codeMirrorEditor.getValue())
             " 
-            x-on:keydown="validateYAML"
+            x-on:keydown.debounce="validateYAML"
```

Comment:
This is so there is a delay after keys of 250ms. Otherwise, this keeps firing while you write and can send a lot of data to the validation endpoint which is not nice. Having this means that after the last keydown has been sent and no other one key has been pressed in 250ms then event fires. This was causing a lot of calls to the validate endpoint while writing (one per key!)